### PR TITLE
Switch all LESS files to SCSS

### PR DESCRIPTION
**Pull request by wfortin**
Thanks to [less2sass](https://www.npmjs.com/package/less2sass)
Had some cleaning up to do a little after, but everything seems fine !

### CODE REVIEW

File: scss/common/common.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import (reference) 'font';
```
Comment (marilynechan) [inline scss/common/common.scss]: This was detected as a file rename/move, but not colors.less to colors.scss. Please do a hg rename for history purpose!

Comment (pastjean) [inline scss/common/common.scss]: hg :trollface:

**Comment by marilynechan** [comment]
All looks good to me, except that a few files do not seem to have been renamed.

**Comment by JPLachance** [comment]
Look's good to me!

**Pull request merged**